core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/Profiler.scala

```diff
@@ -125,6 +142,12 @@ class Profiler(hadoopConf: Configuration, appArgs: ProfileArgs, enablePB: Boolea
       }
     }
     progressBar.foreach(_.finishAll())
+    
+    // Write status reports for all event logs
+    val profileOutputWriter = new ProfileOutputWriter(outputDir, Profiler.PROFILE_LOG_NAME,
+      numOutputRows, outputCSV = outputCSV)
+    val reportResults = generateStatusProfResults(appStatusReporter.asScala.values.toSeq)
+    profileOutputWriter.write("Profiling Status", reportResults)
```

Comment:
There is a tricky difference between Qualification and Profiler.
For instance, Profiler generates a `profile.log` that contains text formatted data for each table.
Therefore, the above code will create two files `profile.log` and `profiling_status.csv`. That's going to cause confusion of the purpose of those files that have naming overlap with the subdirectories of the apps. We need only the CSV file.

Reply:
removed `profile.log` generation by separating `writeCSVTable` as an individual function